facebook/OpenBIC
# fby4: ff: Revise the target address in I2C target init-config table

# Description:
The target address in I2C target init-config table should be the address of BIC.

# Motivation:
Correct the target address so that ASIC could send I2C trace back to FF BIC.

# Test Plan:
1. Check ASIC could write I2C trace back to FF BIC - pass.
2. Check the EID of ASIC on FF have listed under the MCTP service - pass.

# Test Log:

1. Check ASIC could write I2C trace back to FF BIC
- ASIC console I: BONG! I: offset 0x710000 0x40c, data_size 0x70 I: REQ I: 64 0f 0a 41 01 00 29 c8 |d..A..).
	I: 00 80 01 00 2b 72       |....+r
	I: RESP
	I: 40 0f 0c 41 01 29 2b c0 |@..A.)+.
	I: 00 00 01 00 00 2b 00 f3 |.....+..
	I: Wrote 16 bytes to 0x20

2. Check the EID of ASIC on FF have listed under the MCTP service
- ASIC EID is 0x43 root@bmc:~# busctl tree xyz.openbmc_project.MCTP `- /xyz `- /xyz/openbmc_project `- /xyz/openbmc_project/mctp `- /xyz/openbmc_project/mctp/1 |- /xyz/openbmc_project/mctp/1/40 |- /xyz/openbmc_project/mctp/1/41 |- /xyz/openbmc_project/mctp/1/43 `- /xyz/openbmc_project/mctp/1/8
